card/lib/card/view/stub.rb

```diff
@@ -11,7 +11,7 @@ class View
     module Stub
       # @return [String]
       def stub
-        "<card-view>#{stub_json}</card-view>".html_safe
+        "(stub)#{stub_json}(/stub)".html_safe
```

Comment:
FYI, this is our new stub syntax.  We should probably discuss.

Reply:
I think that violates html specs. Custom tags are supposed to have a "-" in the name.

Reply:
oh, it's not even a tag anymore.

Reply:
I guess if I write "(stub)" in a card everything goes to hell. Isn't it better to change the tag strip algorithm so that it doesn't remove custom tags?

Reply:
let's discuss.  the strip tag solution was my first impulse until I saw that there were lots of places where there was stripping going on using several different mechanisms, and I didn't want to update them all.  maybe I'll change it to something slightly less likely to be stumbled upon